source/find_getmore_killcursors_commands.rst

```diff
@@ -356,6 +347,12 @@ The new **find** command has different semantics to the existing 3.0 and earlier
 
 Once the limit on the cursor has been reached the server will destroy the cursor and return a **cursorId** of **0** in the **OP_REPLY**. This differs from existing **OP_QUERY** behavior where there is no server side concept of limit and where the driver **MUST** keep track of the limit on the client side and **MUST** send a **OP_KILLCURSORS** wire protocol message when it limit is reached.
 
+When using the **find** and **getMore** command the same **batchSize** value **SHOULD** be passed to both commands. This means that if the **find** command had a **batchSize** of `3` the subsequent **getMore** commands **SHOULD** also have **batchSize** set to `3`. 
```

Comment:
I really think this "should" should be a MUST. I don't see any good reason to allow drivers to diverge in implementation here. If they can't actually do it in their codebase for whatever reason, then they'll be out of spec until they fix it. Not the end of the world, but certainly something for them to come back to and refactor.

Do you have a good reason for this allowance?